### DISCUSSION TITLE
OCaml 5.0 support

### DISCUSSION THREAD

**Discussion by 2BitSalute** [ocamllabs/vscode-ocaml-platform#952] [Q&A]
Hi, could I reasonably get the OCaml Platform extension to work with OCaml 5.0? It looks like I would currently have to downgrade to 4.12 to use it, but I specifically want to try 5.0 for some new features.

**Comment by rgrinberg** [comment]
As far as I know, there's nothing in the platform extension that prevents it from working with 5.0. The only thing you need is a 5.0 compatible LSP server.

**Comment by ulugbekna** [comment]
You can get an OCaml 5.0-compatible ocaml-lsp-server by running 

```
opam pin -y git+https://github.com/patricoferris/ocaml-lsp#dc9a4ef8529628fe023e1ed034ffe6b517ea4f1a
```

A little more context to rgrinberg's answer above: this repo is for the vscode OCaml extension, which will works regardless of ocaml version the user has. To have this extension work properly with your ocaml 5.0 codebase, you just need a version of ocaml-lsp-server that supports 5.0. You can get it by running the command above. 

Apologies for a belated reply; missed that notification email :-)
